src/www/views/servers_view/index.ts

```diff
@@ -14,139 +14,155 @@
-            <div
-              class="footer subtle"
-              inner-h-t-m-l="[[localize('server-create-your-own-zero-state-access', 'breakLine', '<br/>', 'openLink', '<a href=https://s3.amazonaws.com/outline-vpn/get-started/index.html#step-1>', 'openLink2', '<a href=https://www.reddit.com/r/outlinevpn/wiki/index/outline_vpn_access_keys/>', 'closeLink', '</a>')]]"
-            ></div>
-          </template>
-        </div>
-      </template>
-      <user-comms-dialog
-        id="autoConnectDialog"
-        localize="[[localize]]"
-        title-localization-key="auto-connect-dialog-title"
-        detail-localization-key="auto-connect-dialog-detail"
-        fire-event-on-hide="AutoConnectDialogDismissed"
-      ></user-comms-dialog>
-      <template is="dom-if" if="[[!shouldShowZeroState]]">
-        <server-list id="serverList" servers="[[servers]]" localize="[[localize]]"></server-list>
-      </template>
-    </div>
-  `,
+    `,
+  ];
+
+  @property({type: Function}) localize: Localizer = msg => msg;
+  @property({type: Boolean}) shouldShowAccessKeyWikiLink = false;
+  @property({type: Array}) servers: ServerListItem[] = [];
 
-  is: 'servers-view',
+  get shouldShowZeroState() {
+    return this.servers ? this.servers.length === 0 : false;
+  }
 
-  properties: {
-    localize: Function,
-    useAltAccessMessage: Boolean,
-    servers: Array,
-    shouldShowZeroState: {
-      type: Boolean,
-      computed: '_computeShouldShowZeroState(servers)',
-    },
-  },
+  private requestPromptAddServer() {
+    this.dispatchEvent(new CustomEvent('add-server', {bubbles: true, composed: true}));
+  }
 
-  _computeShouldShowZeroState(servers: ServerListItem[]) {
-    return servers ? servers.length === 0 : false;
-  },
+  private get zeroStateContent(): DirectiveResult<typeof UnsafeHTMLDirective> {
+    let msg;
+    if (this.shouldShowAccessKeyWikiLink) {
+      msg = this.localize(
+        'server-create-your-own-zero-state-access',
+        'breakLine', '<br/>',
+        'openLink', '<a href=https://s3.amazonaws.com/outline-vpn/get-started/index.html#step-1>',
+        'openLink2', '<a href=https://www.reddit.com/r/outlinevpn/wiki/index/outline_vpn_access_keys/>',
+        'closeLink', '</a>');
+    } else {
+      msg = this.localize(
+        'server-create-your-own-zero-state',
+        'breakLine', '<br/>',
+        'openLink', '<a href=https://s3.amazonaws.com/outline-vpn/get-started/index.html#step-1>',
+        'closeLink', '</a>');
+    }
+    return unsafeHTML(msg);
+  }
 
-  _requestPromptAddServer() {
-    this.fire('PromptAddServerRequested', {});
-  },
-});
+  render() {
+    if (this.shouldShowZeroState) {
+      return html`
+        <button type="button" @click=${this.requestPromptAddServer}>
+          <server-connection-indicator connection-state="disconnected"></server-connection-indicator>
+          <header>
+            <h1>${this.localize('server-add')}</h1>
```

Comment:
`<h1>` here is used explicitly in the context of `<header>` that's heading sectioned content. It would be odd to have the first h* element inside `<header>` be `<h2>`. Although as I write this, I realize I'm actually missing a sectioning element here. Added that.

Reply:
The primary topic of this page is not "Outline" though, it doesn't help accessibility because all pages would be the same. The "Add server" is the appropriate (though clunky) topic of the page, even setting aside that you can have multiple section headers. The source you link gives a good example and also admits it further down:

> **Can a page have multiple H1 tags?**
> [...] In some circumstances, where a page truly covers multiple high-level topics, multiple H1s can be used. Google’s John Muller has stated that it will not penalize pages with multiple H1 tags. [...]

The only argument I could see is that "Add server" shouldn't be a header at all, but just regular content, in which case we should revert to a text node or `<p>` element. Maybe "Servers" should be the appropriate heading for both the server list and the empty state?